### PR TITLE
Fix #1482: InputPhone inputStyleClass

### LINKED ISSUE #1482: InputPhone: Not respecting inputStyleClass
### Describe the bug

using `inputStyleClass="test"` you will not see test appear on the input.

### Reproducer

_No response_

### Expected behavior

class should be applied

### PrimeFaces Extensions version

13.0.7

### JSF implementation

All

### JSF version

2.3

### Browser(s)

_No response_

### PR DESCRIPTION
**Pull request by melloware**
Fix #1482: InputPhone inputStyleClass

### CODE REVIEW

File: core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java [hunk 1]
```diff
@@ -149,13 +149,14 @@ protected void encodeInput(final FacesContext context, final InputPhone inputPho
...
         final String inputId = clientId + "_input";
         final String inputStyle = inputPhone.getInputStyle();
+        final String styleClass = createStyleClass(inputPhone, "inputStyleClass", InputText.STYLE_CLASS);
 
         writer.startElement("input", null);
         writer.writeAttribute("id", inputId, null);
         writer.writeAttribute("name", inputId, null);
         writer.writeAttribute("type", inputPhone.getType(), null);
         writer.writeAttribute("value", valueToRender, null);
-        writer.writeAttribute(Attrs.CLASS, createStyleClass(inputPhone, InputText.STYLE_CLASS), "styleClass");
```
Comment (NicolaIsotta) [inline core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java:158]: Shouldn't the third param be `"inputStyleClass"` (like line 152)?

File: core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java [hunk 2]
```diff
@@ -149,13 +149,14 @@ protected void encodeInput(final FacesContext context, final InputPhone inputPho
...
+        writer.writeAttribute(Attrs.CLASS, styleClass, "styleClass");
```
Comment (NicolaIsotta) [inline core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java:159]: Shouldn't the third param be `"inputStyleClass"` (like line 152)?

Comment (melloware) [reply core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java:159]: Yeah I guess it should be!

Comment (melloware) [reply core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java:159]: Fixed!

**Pull request merged**